docs/development.rst

```diff
@@ -285,35 +397,77 @@ HMC vault file
 ^^^^^^^^^^^^^^
 
 The HMC vault file specifies credentials for real HMCs to be used for any
-code that uses the :ref:`zhmcclient.testutils module` such as the end2end
-tests or the examples in the `examples directory`_.
+code that uses the :mod:`zhmcclient.testutils` module, such as the end2end tests,
+the `example scripts`_, other zhmcclient projects, or even projects by users of
```

Comment:
Those "_" are on purpose?

Reply:
Yes, they are RST syntax for generating a link to that target.